### ISSUE TITLE
Feature Request: A way to see "home" suggestions even with redirect on.

### ISSUE THREAD

**Issue by JasonJAyalaP** [lawrencehook/remove-youtube-suggestions#89]
I use the redirect home page to watch later/subscriptions feature. However, there are times when I'm curious about the suggestions on "home". But with a redirect on, there is no way to manually view home. 

Yes, I know this addon is called remove suggestions! Sorry for the awkward request!

Perhaps a (sub?)option in the redirect menu for "Don't redirect home link in left navbar."?

**Comment by lawrencehook** [comment]
Good idea! I've encountered the same thing when I have redirect enabled. I'll see what I can do.
